sir_cake/store/utils.py

```diff
@@ -0,0 +1,79 @@
+from django.contrib import messages
+from django.core.exceptions import ObjectDoesNotExist
+
+from seller.models import Item, OrderPackage
+from users.models import AnonymousUser
+from .models import Basket
+
+
+def add_one_item_basket(request, context):
+    # Anonymous user without session
+    if request.session.session_key is None:
+        return 'continue_purchase'
+
+    basket = []
```

Comment:
There is no need to initialize `basket` variable here because it will get initialized either in if or in else for sure